service/s3control/deserializers.go

```diff
@@ -89,7 +89,7 @@ func awsRestxml_deserializeOpErrorCreateAccessPoint(response *smithyhttp.Respons
 	errorCode := "UnknownError"
 	errorMessage := errorCode
 
-	errorComponents, err := s3shared.GetErrorResponseComponents(errorBody)
+	errorComponents, err := s3shared.GetErrorResponseComponents(errorBody, response.StatusCode)
```

Comment:
Should s3 control have this customization behavior to, or just S3?

Reply:
just s3